app/services/project_import.rb

```diff
@@ -21,10 +21,6 @@ def run
         output = []
         mediaflux_projects = CSV.new(csv_data, headers: true, liberal_parsing: true)
         mediaflux_projects.each do |project_metadata|
-          # skip projects not part of the current namespace in dev & test mode since we have both mediaflux instances in one server
```

Comment:
With the `tigerdata.project.create` service now all the projects (test and dev) are saved on the same root `/princeton/tigerdata` and therefore this logic didn't work anymore. We could prefix all test projects to be saved somewhere else (e.g. `/princeton/tigerdata/rspec-projects/`) but I didn't do that (PR is already too large) so I commented this logic and adjusted the tests to look inside the result to make sure the right projects are created.